tools/changelog-generator/changelog.py

```diff
@@ -0,0 +1,103 @@
+import requests
+import os
+from datetime import datetime, timedelta
+
+
+base_url = 'https://api.github.com'
+repo = 'Kong/docs.konghq.com'
+current_directory = os.getcwd()
+file_path = os.path.join(current_directory, 'docs_changelog.md')
+
+
+today = datetime.now()
+last_week = today - timedelta(days=7)
+
+since = last_week.date().isoformat()
+until = today.date().isoformat()
+
+#api_key = 'insert API key'
+api_key = os.getenv('API_KEY')
+
+pr_url = f'{base_url}/repos/{repo}/pulls?state=closed&base=main&sort=updated&direction=desc&since={since}&until={until}'
```

Comment:
```suggestion
pr_url = f'{base_url}/repos/{repo}/pulls?state=merged&base=main&sort=updated&direction=desc&since={since}&until={until}'
```
`closed` is pulling in PRs that were closed but not merged, eg https://github.com/Kong/docs.konghq.com/pull/5661 

( You can see it in your generated file here:https://github.com/Kong/docs.konghq.com/pull/5825/files#diff-addf326bfd6a7419476ea244ac1e9841b4f49cd692510e20a5936340748949f4R29)